device-types/Quantum/Xcellis-WFDirector-Gen2-12G.yaml

```diff
@@ -0,0 +1,42 @@
+---
+manufacturer: Quantum
+model: Xcellis Workflow Director Gen2 12G
+slug: quantum-xcellis-workflow-director-gen2-12g
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: ServicePort
+    type: rj-45
+power-ports:
```

Comment:
I'm guessing these are the controller devices, as these are child devices they don't have their own power ports.

Reply:
Ah okay, yes just convert them to `module-bays`